syntax = "proto3";

message gas_online_count_notify {
	uint32 count = 1;
}